ios/RNWatch/RNWatch.m

```diff
@@ -36,6 +36,7 @@
 static NSString *EVENT_SESSION_BECAME_INACTIVE = @"WatchSessionBecameInactive";
 static NSString *EVENT_PAIR_STATUS_CHANGED = @"WatchPairStatusChanged";
 static NSString *EVENT_INSTALL_STATUS_CHANGED = @"WatchInstallStatusChanged";
+static NSString *EVENT_WATCH_SEND_ERROR = @"WatchSendError";
```

Comment:
I think it might be better to have seperate events for errors from user info & application context?

Reply:
e.g. something like `EVENT_WATCH_USER_INFO_ERROR` or `EVENT_WATCH_APPLICATION_CONTEXT_ERROR` or something like that

Reply:
Done.